src/Covid19Dashboard/IllinoisMapChart/index.jsx

```diff
@@ -1,49 +1,31 @@
+/* eslint-disable react/sort-comp */
```

Comment:
minor: is it necessary to disable this?

Reply:
no its not, good catch i will remove and lint